websites/api-docs-smoke-test/source-raml/test/api.raml

```diff
@@ -147,33 +162,40 @@ uses:
         uriParameterOne:
           type: string
           description: A Path parameter in the equal-sign-pseudo-query notation
+          example: lastName = "Doe"
         uriParameterTwo:
           type: string
           description: A Path parameter as a plain variable path element
+          example: plain
       post:
         description: Updates the resource. OutOfOrderPropertiesTestType is used because it is big and long.
         queryParameters:
           queryParameterOne:
             type: number
             description: some parameter to construct a combination of URI and query params
             required: false
+            example: 1
           queryParameterTwo:
             type: string
             required: false
+            example: string-path
```

Comment:
For testing an implementation, it would be good to have a test case where one optional query parameter has no example set.   maybe this one?